kikuchipy/signals/ebsd.py

```diff
@@ -1072,7 +1072,9 @@ def average_neighbour_patterns(
             averaging_window = copy.copy(window)
         else:
             averaging_window = Window(
-                window=window, shape=window_shape, **kwargs,
+                window=window,
```

Comment:
Actually, it might be MY black that previously didn't put these parameters on separate lines when the last parameter has a comma after it... If you find that the steps above still wants to put these on separate lines, let's just leave these changes in this PR, and I'll run a Black formatting PR for the other lines...